djangomodelimport/resultset.py

```diff
@@ -14,10 +15,15 @@ def __init__(self, headers, header_form):
         self.header_form = header_form
 
     def __repr__(self):
-        return f'ImportResultSet ({len(self.results)} rows, {len(self.get_errors())} errors)'
+        i = len(self.results)
+        j = len(self.get_errors())
+        k = len(self.get_warnings())
+        return f"ImportResultSet ({i} rows, {j} errors, {k} warnings)"
```

Comment:
lol. maybe `str.format()` makes more sense, or just naming them `n_rows`, `n_errors`, `n_warnings`.
Though i guess `i, j, k` does make it clear to the developer that these variables are super temporary.

Not fussed if it says as is.